### PR TITLE
Fixed Grammar and Typo mistakes

### PR DESCRIPTION
**Pull request by steffinstanly**

### CODE REVIEW

File: Add-ons/UmbracoCourier/Architecture/DataResolvers.md [hunk 1]
```diff
@@ -1,159 +1,157 @@
...
-	    <!-- add new datatype elements for data types that stores files as a path ex: /meda/223/file.png -->
-	    <add key="Upload">5032a6e6-69e3-491d-bb28-cd31cd11086c</add>
-	    <add key="Ucomponents-Filepicker">318a9c2e-3966-4979-8c1d-575c5d5f669b</add>
-	</files>
-
-### UsercontrolWrapper
-* **Full name:** `Umbraco.Courier.DataResolvers.UsercontrolWrapper`
-* **Triggers on:**  The UsercontrolWrapper Data types
-*If the data type has a .ascx file selected as render, the file is added as a resource to ensure it is transferred with the data type.
+
```
Comment (jmayntzhusen) [inline Add-ons/UmbracoCourier/Architecture/DataResolvers.md]: Seems you have simply removed the headlines and explanation here, not sure why? 🙂

File: Add-ons/UmbracoCourier/Architecture/DataResolvers.md [hunk 2]
```diff
@@ -1,159 +1,157 @@
...
-	<files>
-	    <!-- add new datatype elements for data types that stores files as a path ex: /meda/223/file.png -->
-	    <add key="Upload">5032a6e6-69e3-491d-bb28-cd31cd11086c</add>
-	    <add key="Ucomponents-Filepicker">318a9c2e-3966-4979-8c1d-575c5d5f669b</add>
-	</files>
-
-### UsercontrolWrapper
-* **Full name:** `Umbraco.Courier.DataResolvers.UsercontrolWrapper`
-* **Triggers on:**  The UsercontrolWrapper Data types
-*If the data type has a .ascx file selected as render, the file is added as a resource to ensure it is transferred with the data type.
```
Comment (jmayntzhusen) [inline Add-ons/UmbracoCourier/Architecture/DataResolvers.md]: You removed another explanation here, why is that? 🙂

Comment (steffinstanly) [reply Add-ons/UmbracoCourier/Architecture/DataResolvers.md]: sorry my mistake. i have rectified it now

**Comment by jmayntzhusen** [comment]
Hey @steffinstanly 

Thanks a lot for the fixes 😃

Do you have a profile on https://our.umbraco.com? we give all contributors a badge there!

**Comment by nul800sebastiaan** [comment]
Hi there @steffinstanly,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**